configs/projects/puppet-agent.rb

```diff
@@ -82,8 +121,26 @@
   end
 
   # Platform specific
-  proj.setting(:cflags, "-I#{proj.includedir} -I/opt/pl-build-tools/include")
-  proj.setting(:ldflags, "-L#{proj.libdir} -L/opt/pl-build-tools/lib -Wl,-rpath=#{proj.libdir}")
+  if platform.is_windows?
+    if platform.architecture == "x64"
+      arch = "64"
+      proj.setting(:cflags, "-IC:/tools/pl-build-tools/include -IC:/tools/mingw#{arch}/include -IC:/tools/mingw64/x86_64-w64-mingw32/include -I#{platform.convert_to_windows_path(proj.includedir)}")
```

Comment:
Why do we need mingw64/x86_64-w64/mingw32/include? GCC should already know about that location.